test/Syntax/diagnostics_verify.swift

```diff
@@ -19,3 +19,7 @@ class { // expected-error {{unknown declaration syntax exists in the source}}
   // expected-error@-4 {{top-level statement cannot begin with a closure expression}}
 
 }
+
+#if swift(<1)
+print("Wat")
```

Comment:
Am I mistaken, or does this really not actually verify anything?  It would require that the user manually verify that the inactive region is processed.

Reply:
This verifies we can get syntax nodes for inactive code like `print("wat")`, otherwise an error message will be emitted to complain about unknown syntax nodes.

Reply:
How about we add an extra step to sanity check that everything is parsed and round-tripped ?
E.g. add a `RUN` line to check that printing the syntax-tree will result in the same output as the input source file.
This will ensure that everything is parsed as expected.